Tests/ApiVideoPlayerTests/IntegrationTests/ApiVideoPlayerControllerIntegrationTests.swift

```diff
@@ -0,0 +1,115 @@
+@testable import ApiVideoPlayer
+import CoreMedia
+import XCTest
+
+@available(iOS 14.0, *)
+final class ApiVideoPlayerControllerIntegrationTests: XCTestCase {
+  private let validVideoId = "vi2G6Qr8ZVE67dWLNymk7qbc"
+  private let invalidVideoId = "unknownVideoId"
+
+  func testValidVideoIdPlay() throws {
+    let expectation = self.expectation(description: "delegate is called 2 times")
+    expectation.expectedFulfillmentCount = 2
+
+    let events = PlayerEvents(
+      didPrepare: { () in
+        print("ready")
+        expectation.fulfill()
+      },
+      didPlay: { () in
+        print("play")
+        expectation.fulfill()
+      }
+    )
+    let controller = ApiVideoPlayerController(videoId: validVideoId, videoType: .vod, events: events)
+    controller.play()
+    waitForExpectations(timeout: 10, handler: nil)
```

Comment:
Use multiple expectation. Same as in `ApiVideoPlayerViewIntegrationTests`'s `testValidVideoIdPlay`